src/providers/ffz/FfzEmotes.cpp

```diff
@@ -222,7 +222,7 @@ void FfzEmotes::loadChannel(
             auto shared = channel.lock();
             if (!shared)
                 return;
-            if (result.status() == 203)
+            if (result.status() == 203 || result.status() == 404)
```

Comment:
Before #2304 was merged in, we had inconsistent error codes, thus according to [enum `QNetworkReply::NetworkError`](https://doc.qt.io/qt-5/qnetworkreply.html#NetworkError-enum), HTTP status code `404` used to be `203`, however after merging #2304, we no longer ever return `203`, so I believe this can be simplified
```suggestion
            if (result.status() == 404)
```